clients/fossid-webapp/src/main/kotlin/model/identification/common/LicenseMatchType.kt

```diff
@@ -20,5 +20,5 @@
 package org.ossreviewtoolkit.clients.fossid.model.identification.common
 
 enum class LicenseMatchType {
-    SNIPPET, FILE
+    FILE, SNIPPET
```

Comment:
One last nit: I like the fact that you did wrap the last enum, and I believe we should then wrap the lines in this enum, too, to consistently have them all wrapped.